firmware/include/cc2531.h

```diff
@@ -46,7 +46,12 @@ SFRX(USBF5,     0x622A);        // Endpoint 5 FIFO
 // USB activities
 #define USB_ENABLE_PIN              P1_0
 //#define USB_ENABLE_PIN              P1_1
+
+// NOP macro which may be defined elsewhere
+#ifndef NOP
```

Comment:
is there a central location that would work here?  global.h, assuming proper order of includes?  i hate the duplication and would avoid it if possible.
does the code compile without this one? (assuming not, but i have to ask)